translations/ui-licenses/en.json

```diff
@@ -99,6 +99,7 @@
   "settings.terms.newLicenseTerm": "New license term",
   "settings.terms.callout.delete.error": "There was an error deleting the term. {error}",
   "settings.terms.callout.delete.success": "Term successfully deleted.",
+  "settings.terms.callout.delete.termInUse": "<strong><span class='{className}'>Error:</span> Term</strong> was not deleted because it is in use on one or more licenses.",
```

Comment:
I realise you're just following the mockup, but this design actually breaks with Folio convention for showing errors within callouts. 

- The fact that it's an error is intended to be communicated by the styling of the callout (via the `type: 'error',` you set) rather than an extra styled "Error: " in it.
- Significantly, I'm pretty sure the red-on-red is gonna fail an a11y check for text-to-background contrast.
- Finally and only minorly, we're putting ourselves at greater risk from future changes to the Callout component by restating what it may state in the future. Eg, we're rendering an "error" callout, its styling could easily and reasonably be changed in the future to say "Error" itself. Then we say "Error" twice. 😬 

I'm pinging Gill with a link to this comment because I'd confirmation on this pattern before we merge it.

Reply:
I fully agree with the above